### DISCUSSION TITLE
OpenSSL 3.1 support

### DISCUSSION THREAD

**Discussion by hchu-alvaria** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#1110] [Q&A]
Looks like the AWS KVS C++ Producer SDK still depends on OpenSSL 1.1.1t, which reached EOL on September 11, 2023. Is there a plan to update it to use OpenSSL 3.1?

Thanks.

**Comment by disa6302** [comment]
@hchu-alvaria ,

We do have plans of updating, but no ETA yet. Will update this thread when we have an ETA.
